### ISSUE TITLE
Add Russian translation for Real Weapon Names optional addon

### ISSUE THREAD

**Issue by SurDno** [Decane/SRP#83]
olegj5 and I did a thing: https://yadi.sk/d/iUsfEya9ZLpkXg

**Comment by Decane** [comment]
>```
>	<string id="st_wpn_spas12">
>		<text>СПСА-14</text>
>	</string>
>```
Did you accidentally upload the wrong file? Isn't "СПСА-14" [the original Russian 'fake' name](https://stalker.fandom.com/ru/wiki/%D0%A1%D0%9F%D0%A1%D0%90-14) of the SPAS-12? The other weapons also seem to have their original 'fake' names.

(My copy of CS doesn't come with the Russian version of st_items_weapons.xml so I can't compare to see what's changed from vanilla in the file you uploaded.)

**Comment by SurDno** [comment]
I actually did somehow.

Also, I thought that this is the file that was made by olegj5 and me during localization changes, but it turned out it just was in my gamedata, probably during some merge with SRP. It may be OGSM or Mystery. I checked changes through WinMerge though and it turned out that everything is right.

How did you change the modified weapons' names though? In this file they all use names like "Normal_weapon_name (modified)" or "Normal_weapon_name (sniper)" since most of these modifications never existed outside of S.T.A.L.K.E.R. universe and your names are far less real. I may change this to direct translation of your upgrade names if you insist.

Also: are you willing to fix #12? I can add strings 7H1 and change 7H14 from 7H1 to 7H14, like you did in the English localization, but I'm not sure this in the scope of the SRP. This has not been touched in this file (yet).

If you're willing to add this as is and don't mind some inconsistencies between localizations, here's the updated file (hope now I haven't messed up anything): https://yadi.sk/d/oxqp38wDCEvHOA . If you have any suggestions about how to improve this, how should I deal with upgrade names and what to do with 7.62, PM me.
